src/tests/ftest/util/server_utils_params.py

```diff
@@ -269,6 +289,31 @@ def update_log_files(self, control_log, helper_log, server_log):
                     "".join(log_name),
                     "server_config.server[{}].log_file".format(index))
 
+    def get_interface_envs(self, index=0):
+        """Get the environment variable names and values for the interfaces.
+
+        Args:
+            index (int, optional): server index from which to obtain the
+                environment variable values. Defaults to 0.
+
+        Returns:
+            EnvironmentVariables: a dictionary of environment variable names
+                and their values extracted from the daos_server yaml
+                configuration file.
+
+        """
+        env = EnvironmentVariables()
+        mapping = {
+            "OFI_INTERFACE": "fabric_iface",
+            "OFI_PORT": "fabric_iface_port",
+            "CRT_PHY_ADDR_STR": "provider",
+        }
+        for key, name in mapping.items():
+            value = self.server_params[index].get_value(name)
+            if value is not None:
+                env[key] = value
+
+        return env
```

Comment:
Why is this being added? It looks like it came from an outdated copy of the file.